lib/modules/datasource/aws-rds/readme.md

```diff
@@ -77,3 +81,30 @@ spec:
   # amiFilter=[{"Name":"engine","Values":["mysql"]},{"Name":"engine-version","Values":["5.7"]}]
   engineVersion: 5.7.34
 ```
+
+#### Using Terraform, `aws-rds` datasource and Aurora MySQL
+
+Here is the Renovate configuration to use Terraform, `aws-rds` and Aurora MySQL:
+
+```json
+{
+  "customManagers": [
+    {
+      "description": "Update RDS",
+      "customType": "regex",
+      "fileMatch": [".+\\.tf$"],
+      "matchStrings": [
+        "\\s*#\\s*renovate:\\s*amiFilter=(?<lookupName>.+?) depName=(?<depName>.*) versioning=(?<versioning>.*)\\s*.*_version\\s*=\\s*\"(?<currentValue>.*)\""
+      ],
+      "datasourceTemplate": "aws-rds"
+    }
+  ]
+}
+```
+
+The above configuration will ...insert explanation of bits below...
```

Comment:
Need a maintainer suggestion for this line too. 😉

Reply:
Where did this code come from?

Reply:
I copy/pasted it from:

- https://github.com/renovatebot/renovate/discussions/19928#discussioncomment-9868352